src/ontoweaver/transformer.py

```diff
@@ -46,22 +47,38 @@ def __call__(self, row, i):
 
 
 class split(base.Transformer):
-    """Transformer subclass used to split cell values at defined separator and create nodes with
+    """Transformer subclass used to split cell values at defined separator and label_maker nodes with
     their respective values as id."""
 
-    def __init__(self,properties_of, branching_properties = None, columns=None, output_validator: validate.OutputValidator = None, raise_errors = True, **kwargs):
+    class ValueMaker(make_value.ValueMaker):
+
+        def __init__(self, raise_errors: bool = True, separator: str = None):
+            self.separator = None
+            super().__init__(raise_errors)
+
+        def select(self, columns, row, i):
+
+            for key in columns:
+                items = str(row[key]).split(self.separator)
+                for item in items:
+                    yield item
+
+    def __init__(self, properties_of, value_maker = ValueMaker(), label_maker = None, branching_properties = None, columns=None, output_validator: validate.OutputValidator = None, raise_errors = True, **kwargs):
         """
         Initialize the split transformer.
 
         Args:
             properties_of: Properties of the node.
+            value_maker: the ValueMaker object used for the logic of cell value selection for each transformer.
+            label_maker: the LabelMaker object used for handling the creation of the output of the transformer. Default is None.
             branching_properties: in case of branching on cell values, the dictionary holding the properties for each branch.
             columns: The columns to be processed.
             sep: Character(s) to use for splitting.
             output_validator: the OutputValidator object used for validating transformer output.
             raise_errors: if True, the caller is asking for raising exceptions when an error occurs
         """
-        super().__init__(properties_of, branching_properties, columns, output_validator, raise_errors = raise_errors, **kwargs)
+        super().__init__(properties_of, value_maker, label_maker, branching_properties, columns, output_validator,
+                         raise_errors=raise_errors, **kwargs)
```

Comment:
… followed by an explicit initialization of the corresponding member variable…
```suggestion
                         raise_errors=raise_errors, **kwargs)
                         if not separator: # Neither empty string or None
                             self.error(f"The `separator` paramater of the `{self.__name__}` transformer cannot be an empty string.")
                         self.separator = separator
```